src/track/video-track.js

```diff
@@ -51,6 +58,24 @@ export default class VideoTrack extends Track {
     return this._height;
   }
 
+  /**
+   * Setter for the qualityLabel of the track.
+   * @public
+   * @param {string} value - The qualityLabel of the track.
+   */
+  set qualityLabel(value: string) {
```

Comment:
why add `qualityLabel`? it already has a label - just remove the label setting from the adapters for video track object.
It will remain null until it is set by application.
See review in playkit-js-ui that accompanies this one.